edenai_apis/apis/amazon/amazon_image_api.py

```diff
@@ -169,9 +176,9 @@ def image__face_detection(
                 if emo.get("Type"):
                     if emo.get("Type").lower() == "happy":  # normalise keywords
                         emo["Type"] = "happiness"
-                    emotion_output[
-                        emo.get("Type").lower()
-                    ] = standardized_confidence_score(normalized_emo / 100)
+                    emotion_output[emo.get("Type").lower()] = (
```

Comment:
Just a small details, but I would have write: emo["Type] instead of emo.get("Type") as we are sure it has a value because there is an IF above